site/layouts/partials/scripts.html

```diff
@@ -18,3 +18,35 @@
 {{- end }}
 
 <script src="{{ $docsJs.Permalink | relURL }}"></script>
+
+<script src="https://cdn.jsdelivr.net/npm/@stackblitz/sdk@1.5.3/bundles/sdk.umd.min.js"></script>
```

Comment:
This URL should be defined in Hugo configuration and use SRI, as for other external dependencies.

Reply:
Done